docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md

```diff
@@ -131,9 +138,9 @@ Building your admin UI with development configuration ...
 ::: tip NOTE
 Using the `--quickstart` flag installs Strapi using an [SQLite](https://www.sqlite.org/index.html) database. You may, at any time, leave off the **--flag**, but you need to follow a few configuration steps for your database choice. **You need to have your database choice installed and running locally before creating your project.**
 
-**Note:** An **SQLite** database is an excellent database to use for prototyping and _developing_ Strapi projects. **SQLite** is a light database that ports effortlessly to the other relational databases (**MySQL**, **PostgreSQL**, and **MariaDB**). It is recommended to **develop** with SQLite and to use another relational database (MySQL, PostgreSQL or MariaDB) in production.
+An **SQLite** database is an excellent database to use for prototyping and _developing_ Strapi projects. **SQLite** is a light database that ports effortlessly to the other relational databases (**MySQL**, **PostgreSQL**, and **MariaDB**). It is recommended to **develop** with SQLite and to use another relational database (MySQL, PostgreSQL or MariaDB) in production.
```

Comment:
I actually disagree with this statement, SQLite is good for prototyping but if you are using a specific type of remote database, you should also be using this locally (makes moving test data locally easier to production or staging, likewise when developing you should be pulling a copy of your production database to see how changes affect your production database.)

Reply:
Better :P